routes/index.js

```diff
@@ -1,67 +1,114 @@
-
 const express = require('express')
 const router = express.Router()
 const db = require('../db')
 const mailClient = require('../mail/index.js')
 var bodyParser = require("body-parser");
 router.use(bodyParser.json())
-const { v1: uuidv1 } = require('uuid');
+const { v4: uuidv4 } = require('uuid');
+const { Connection } = require('pg');
 
 
 router.get('/test', function (req,res) {
-    res.send('works for me')
+  res.send('works for me')
 })
 
 router.get('/test-db', function(req,res) {
-    const query = 'Select * from users'
-    db.query(query)
-    .then(results => {
-        res.send(results.rows)
-    })
+  const query = 'Select * from users'
+  db.query(query)
+  .then(results => {
+    res.send(results.rows)
+  })
 })
```

Comment:
Why is the indentation getting changed here? It's too much work to revert, but in the future, try not to change lines that aren't related to your commits; otherwise, it pollutes the version history of the file.